cmd/gateway/s3x/gateway-s3x.go

```diff
@@ -91,6 +89,9 @@ func init() {
 			},
 		},
 	})
+	if err != nil {
```

Comment:
should probably wrap in `if err := minio.RegisterGatewayCommand....` looks a bit cleaner

Reply:
ok